cli/config/config.go

```diff
@@ -316,6 +316,7 @@ func checkConfigFile(workspace string) {
 		}).Fatal("Cannot create workdir for 'op'")
 	}
 
+	log.Info("Creating %s with default values in %s.", fileName, workspace)
```

Comment:
Isn't this line the same as in L327?

Reply:
This is only for testing purposes, but when I did the merge for some reason I did a wrong merge